### ISSUE TITLE
bug report: A hyperlink in ReadMe.md was invalid. 

### ISSUE THREAD

**Issue by Chinese-Xu** [accord-net/framework#910]
In section: With Visual Studio 2015
Please download and install the following dependencies:
VS2013 C++ platform toolset (for compiling some unit tests only)

The link for VS2013 C++ platform toolset was invalid.
Whether this link should refer to "Visual C++ Redistributable Packages for Visual Studio 2013" ?
https://www.microsoft.com/en-gb/download/details.aspx?id=40784

Thanks,
Wiley Xu

**Comment by cesarsouza** [comment]
Hi Wiley,

Thanks a lot for opening the issue! I will do a revision of the required platforms and update the links, thanks for pointing out the current link for the C++ redistributable for VS 2013.

However, may I ask if you have tried to run the project and run the tests without having the VS 2013 redistributable? 

Regards,
Cesar

**Comment by Chinese-Xu** [comment]
Hi Cesar,

Current issue have been fixed .

My local environment is VS 2015 and already have VS 2013 redistribution.  I just flow the guideline to verify the steps.

Currently I blocked by an issue of "build tools for v141 (Platform Toolset = 'v141') cannot be found".  It caused by project "Accord.Video.FFMPEG (GPL)"

I will open a new issue to report and track it.

Thanks,
Wiley Xu
